src/roles/vstat-predeploy/tasks/vcenter.yml

```diff
@@ -40,6 +40,7 @@
       tasks_from: configure-ovftool
     vars:
       management_network: "Management"
+      vmfolder: "{{ vcenter.vmfolder }}"
```

Comment:
This will fail when vmfolder isn't defined. Ansible will throw an undefined variable error on vcenter.vmfolder.